src/Ztm.WebApi.Tests/SqlCallbackRepositoryTests.cs

```diff
@@ -0,0 +1,162 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Xunit;
+using Ztm.Data.Entity.Contexts;
+using Ztm.Data.Entity.Contexts.Main;
+using Ztm.Data.Entity.Testing;
+
+namespace Ztm.WebApi.Tests
+{
+    public class SqlCallbackRepositoryTests
+    {
+        readonly ICallbackRepository subject;
+        readonly IMainDatabaseFactory dbFactory;
```

Comment:
We don't need to use interface due to it is internal to this class. So just `TestMainDatabaseFactory` is better.